pkg/machines/machines.less

```diff
@@ -270,14 +270,6 @@ table.vcpu-detail-modal-right .btn-group.bootstrap-select.dropdown {
     display: inline-block;
 }
 
-/* FIXME: This CSS should be moved upstream */
-.form-text {
-    /* Set size similar to a widget; like control-label, but no alignment */
-    display: inline-block;
-    padding-top: 3px;
-    margin-bottom: 0;
-}
-
```

Comment:
This block of CSS removals seemed to wash out in the rebase party.

When I rebased what I had locally, this showed up... I guess it didn't make it into the previous merge for whatever reason?

(We should probably reword the commit as to not make it confusing with the other one currently titled exactly the same.)

Reply:
Create a new commit please inside this PR.

Reply:
@KKoukiou : I don't understand -- this *is* already a separate commit. But indeed the commit message doesn't match this change and needs to be improved. @garrett : Could it be that your actual change for reformatting the NIC edit dialog got lost during rebase?

Reply:
@martinpitt: Yeah, I removed this code and squashed it with @KKoukiou's commit, as it was related. I guess one of the merges + rebases one of us did removed it. No harm done; I'll just reword the commit. Glad git caught it. :wink:

Reply:
It's my fault, in the form-layout PR I rebased my local changes on top of garrets commits and force pushed, so this change was missing. Just commit message needs to be adjusted and also we can remove the this class reference.
in:
pkg/machines/components/nicEdit.jsx:            <samp className='form-text' id='mac'>